### DISCUSSION TITLE
Weasis support on  windows Snapdragon copilot Lapto

### DISCUSSION THREAD

**Discussion by arunjoshtech** [nroduit/Weasis#590] [General]
Dear weasis team,
Weasis latest version will work on Windows Snapdragon copilot laptop? My self planning for PC upgradation. Kindly help me to find out the compatibility.

**Comment by nroduit** [comment]
For the moment, we're not distributing any Windows ARM packages. Moreover, few companies supply Windows ARM applications, as there are still gaps in tools and frameworks (only Java LTS is distributed by Azul).

The Weasis X86 package can be installed and run in emulation mode. However, I haven't tried it, so I can't guarantee that it will work properly.

It seems a bit strange to buy a great computer and then have most applications run in emulation mode.
